### PR TITLE
lang: Add some string of app context for french.

### PR DESCRIPTION
**Pull request by Zangetsu38**
# About:
- lang: Add some string of app context for french.
refactor some code of app context for compat.

# Result:
- In Commercial app
image

- in homebrew or when no succès load compat db
image

### CODE REVIEW

**Comment by Croden1999** [comment]
Mr. @Zangetsu38, could u change zh/ch? please🙏🏻

<check_app_state>
zh
检查应用状态
ch
檢查應用狀態

<open_state_report>
zh
打开状态报告
ch
開啟狀態匯報

<create_state_report>
zh
创建状态报告
ch
建立狀態匯報

thank u so much

File: lang/ja.xml [hunk 1]
```diff
@@ -36,24 +36,19 @@
 	</about>
 
 	<app_context>
-		<boot>起動</boot>
-		<check_app_compatibility>アプリの互換性を調べる</check_app_compatibility>
 		<copy_vita3k_summary>Vita3K Summaryをコピー</copy_vita3k_summary>
-		<open_issue>Issueを開く</open_issue>
-		<create_issue>Issueを作成する</create_issue>
```
Comment (Zangetsu38) [inline lang/ja.xml:43]: @Croden1999 you know for both here ?

**Comment by Zangetsu38** [comment]
> Mr. @Zangetsu38, could u update zh/ch? please🙏🏻
> 
> <check_app_state> zh 检查应用状态 ch 檢查應用狀態
> 
> <open_state_report> zh 打开状态报告 ch 開啟狀態匯報
> 
> <create_state_report> zh 创建状态报告 ch 建立狀態匯報
> 
> thank u so much

What is ch and zh ? Said traditionnel or simplified, i no understund other

File: lang/ja.xml (hunk 1 above)
Comment (Croden1999) [reply lang/ja.xml:43]: i only understand en/zh-s/zh-t

**Comment by Croden1999** [comment]
> > Mr. @Zangetsu38, could u update zh/ch? please🙏🏻
> > <check_app_state> zh 检查应用状态 ch 檢查應用狀態
> > <open_state_report> zh 打开状态报告 ch 開啟狀態匯報
> > <create_state_report> zh 创建状态报告 ch 建立狀態匯報
> > thank u so much
> 
> What is ch and zh ? Said traditionnel or simplified, i no understund other
zh for zh-s, ch for zh-t

**Comment by Zangetsu38** [comment]
@Croden1999 confirm me if i have good copy poast in good place ? xD

File: lang/ja.xml (hunk 1 above)
Comment (Zangetsu38) [reply lang/ja.xml:43]: @nishinji  you are called here :D

File: lang/en-gb.xml [hunk 2]
```diff
@@ -36,13 +36,12 @@
...
 	<app_context>
-		<boot>Boot</boot>
-		<check_app_compatibility>Check App Compatibility</check_app_compatibility>
+		<check_app_state>Check App state</check_app_state>
 		<copy_vita3k_summary>Copy Vita3K Summary</copy_vita3k_summary>
-		<open_issue>Open Issue</open_issue>
-		<create_issue>Create Issue</create_issue>
-		<update_database>Update Database</update_database>
-		<copy_app_info>Copy App Info</copy_app_info>
+		<open_state_report>Open state report</open_state_report>
```
Comment (nishinji) [inline lang/en-gb.xml:41]: I think `compat report` would be better.

Comment (Zangetsu38) [reply lang/en-gb.xml:41]: > I think `compat report` would be better.

no, is already compat, menu is Compatiblity/Open state report, and state is perfect

**Comment by Croden1999** [comment]
> @Croden1999 confirm me if i have good copy poast in good place ? xD

very good XD 👍 sir

File: lang/ja.xml (hunk 1 above)
Comment (nishinji) [reply lang/ja.xml:43]:
```
<open_state_report>互換性レポートを開く</open_state_report>
<create_state_report>互換性レポートを作成する</create_state_report>
```

File: lang/ja.xml [hunk 3]
```diff
@@ -36,24 +36,22 @@
...
 	<app_context>
-		<boot>起動</boot>
-		<check_app_compatibility>アプリの互換性を調べる</check_app_compatibility>
-		<copy_vita3k_summary>Vita3K Summaryをコピー</copy_vita3k_summary>
-		<open_issue>Issueを開く</open_issue>
-		<create_issue>Issueを作成する</create_issue>
+		<check_app_state>アプリの状態をチェック</check_app_state>
+		<copy_vita3k_summary>Vita3K概要をコピー</copy_vita3k_summary>
+		<open_state_report>状態レポートを開く</open_state_report>
+		<create_state_report>状態レポートを作成</create_state_report>
```
[the remark this answers is not in the document]
Comment (nishinji) [reply lang/ja.xml:42]: i feel uncomfortable with using 状態
replace it to 互換性

[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/ja.xml:42]: no, i said is state, no compatibility, all lang write same, i no want repead word, is already compatiblité

[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/ja.xml:42]: you write me compatiblité, is wrong, is statut
image

[the remark this answers is not in the document]
Comment (Croden1999) [reply lang/ja.xml:42]: Mr.@nishinji, state already includes compat

[the remark this answers is not in the document]
Comment (nishinji) [reply lang/ja.xml:42]: As a translation, 状態レポート is more correct, but it's uncomfortable as Japanese
whichever🤷‍♂️

[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/ja.xml:42]: is statut, no situation :D 
image

[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/ja.xml:42]: in english said, status, is diferente of state, and i want onlyè state, for using like other part of code
image

File: lang/en.xml [hunk 4]
```diff
@@ -36,13 +36,12 @@
...
+		<check_app_state>Check App state</check_app_state>
 		<copy_vita3k_summary>Copy Vita3K Summary</copy_vita3k_summary>
-		<open_issue>Open Issue</open_issue>
-		<create_issue>Create Issue</create_issue>
-		<update_database>Update Database</update_database>
-		<copy_app_info>Copy App Info</copy_app_info>
+		<open_state_report>Open state report</open_state_report>
+		<create_state_report>Create state report</create_state_report>
+		<update_database>Update database</update_database>
+		<copy_app_info>Copy App info</copy_app_infos>
```
Comment (Croden1999) [inline lang/en.xml:44]: here is extra ‘s’

File: lang/ja.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/ja.xml:42]: > Mr.@nishinji, state already includes compat, the status is more reflective of the details

State, no status, is little diferent ^^, all code already write State for label etc
oi have writed all with using french, for "Rapport d'état" so "State report"

File: lang/en.xml (hunk 4 above)
Comment (Zangetsu38) [reply lang/en.xml:44]: oh fail xD

File: lang/ja.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (nishinji) [reply lang/ja.xml:42]: There's no point in discussing it here
it's fine as is.

[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/ja.xml:42]: good thx :)

**Review by bookmist — APPROVED** [review]

**Pull request merged**